cli/parse_args.ts

```diff
@@ -371,8 +371,8 @@ function hasKey(obj: NestedMapping, keys: string[]): boolean {
     o = (get(o, key) ?? {}) as NestedMapping;
   });
 
-  const key = keys[keys.length - 1];
-  return hasOwn(o, key);
+  const key = keys.at(-1);
+  return typeof key !== "undefined" && hasOwn(o, key);
```

Comment:
```suggestion
  return key !== undefined && hasOwn(o, key);
```
Ditto for other instances of checking if a variable doesn't equal `undefined`.